sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -795,16 +797,24 @@ static const char *getDeviceLibExtensionStr(DeviceLibExt Extension) {
     return "cl_intel_devicelib_imf";
   case DeviceLibExt::cl_intel_devicelib_imf_fp64:
     return "cl_intel_devicelib_imf_fp64";
+  case DeviceLibExt::cl_intel_devicelib_bfloat16:
+    return "cl_intel_bfloat16_conversions";
```

Comment:
All the other cases have the names be the exact same as the enum. Is there a reason this one differs?

Reply:
"cl_intel_bfloat16_conversions" is an actual OpenCL Intel extension. The name was chosen by Intel folks working on the OpenCL standard.
The other names don't reflect actual OpenCL extensions. They are used only within this file as extensions just for the purpose of pulling in various libraries for linking with device code.